docs/openttd.6

```diff
@@ -141,11 +147,12 @@ see
 for a full list.
 .It Fl x
 Do not automatically save to config file on exit.
+.It Fl X
+Do not use global folders to search for files.
 .El
 .Sh SEE ALSO
 .Lk https://wiki.openttd.org "Wiki"
 (includes community maintained manual),
-.Lk https://forum.openttd.org "Forum",
```

Comment:
You say this link doesn't exist in your description, but that is not true. That link does (and has for many years) exactly what you expect: redirect you to our forums. If that doesn't work for you, let me know, as it really should :)

Whether to remove it or not, I don't actually care :)

Reply:
Well, I'm sure that the link did not work as it should have when I checked all the links in the man page. It could have been some transient issue though; I didn't investigate much further as I didn't really care whether it still worked or not, just that there weren't broken links in the man page.

Reply:
Just to check: now you know what you know, you still want to leave it out? Again, I don't really care, as long as we do it for the right reasons :D

PS: the singular form as it is here, is correct. Your description mentions `forums.openttd.org`; that subdomain does not exist. Maybe there things broke down :)